python/tvm/runtime/disco/session.py

```diff
@@ -0,0 +1,294 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This module defines a Session in Disco. Session is the primary interface that users interact
+with the distributed runtime.
+"""
+from typing import Any, Callable, Optional, Sequence
+
+from ..._ffi import register_object
+from ..._ffi.runtime_ctypes import Device
+from ..ndarray import NDArray
+from ..object import Object
+from . import _ffi_api
+
+
+@register_object("runtime.disco.DRef")
+class DRef(Object):
+    """An object that exists on all workers. The controller process assigns a unique "register id"
+    to each object, and the worker process uses this id to refer to the object residing on itself.
```

Comment:
```suggestion
    """An object that exists on all workers. The controller assigns a unique "register id"
    to each object, and the worker uses this id to refer to the object residing on itself.
```
remove the word "process" to avoid confusion